packages/ckeditor5-emoji/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@ckeditor/ckeditor5-emoji",
+  "version": "0.0.1",
+  "description": "Emoji feature for CKEditor 5.",
+  "keywords": [
+    "ckeditor",
+    "ckeditor5",
+    "ckeditor 5",
+    "ckeditor5-feature",
+    "ckeditor5-plugin",
+    "ckeditor5-dll"
+  ],
+  "type": "module",
+  "main": "src/index.ts",
+  "dependencies": {
+    "ckeditor5": "43.3.0",
+    "@ckeditor/ckeditor5-core": "43.3.0",
+    "@ckeditor/ckeditor5-typing": "43.3.0",
+    "@ckeditor/ckeditor5-ui": "43.3.0",
+    "@ckeditor/ckeditor5-utils": "43.3.0"
+  },
+  "devDependencies": {
+    "@ckeditor/ckeditor5-cloud-services": "43.3.0",
+    "@ckeditor/ckeditor5-dev-utils": "^45.0.0",
+    "@ckeditor/ckeditor5-easy-image": "43.3.0",
+    "@ckeditor/ckeditor5-editor-classic": "43.3.0",
+    "@ckeditor/ckeditor5-image": "43.3.0",
+    "@ckeditor/ckeditor5-theme-lark": "43.3.0",
+    "typescript": "5.0.4",
+    "webpack": "^5.94.0",
+    "webpack-cli": "^5.1.4"
+  },
```

Comment:
Let's add dependencies when they are really needed.
Emoji feature will not require `@ckeditor/ckeditor5-cloud-services`.